packages/xo-server-sdn-controller/src/private-network/private-network.js

```diff
@@ -57,26 +57,16 @@ export class PrivateNetwork {
     const encapsulation = otherConfig['xo:sdn-controller:encapsulation'] ?? 'gre'
     const vni = otherConfig['xo:sdn-controller:vni'] ?? '0'
     const password = otherConfig['xo:sdn-controller:encrypted'] === 'true' ? createPassword() : undefined
-    const pifDevice = otherConfig['xo:sdn-controller:pif-device']
-    const pifVlan = +otherConfig['xo:sdn-controller:vlan']
-    const hostPif = hostClient.host.$PIFs.find(
-      pif => pif?.device === pifDevice && pif.VLAN === pifVlan && pif.ip_configuration_mode !== 'None'
-    )
-    const centerDevice = centerNetwork.other_config['xo:sdn-controller:pif-device']
-    const centerVlan = +centerNetwork.other_config['xo:sdn-controller:vlan']
-    const centerPif = centerClient.host.$PIFs.find(
-      pif => pif?.device === centerDevice && pif.VLAN === centerVlan && pif.ip_configuration_mode !== 'None'
-    )
+
+    const hostPif = this._getHostTransportPif(host)
```

Comment:
I see that xo:sdn-controller:pif-device is also used in createTunnel , will it create issue there too ? 
same question for xo:sdn-controller:vlan